stix2generator/language/interop_custom.json

```diff
@@ -0,0 +1,309 @@
+{
```

Comment:
Is there a reason this file is under `/language`? Maybe it should be one directory up, next to the default `stix21_registry.json`.

Reply:
moved up on directory